src/main/java/org/broadinstitute/hellbender/tools/copynumber/ModelSegments.java

```diff
@@ -619,12 +630,14 @@ private AllelicCountCollection genotypeHets(final SampleLocatableMetadata metada
 
         logger.info("Genotyping heterozygous sites from available allelic counts...");
 
+        AllelicCountCollection filteredAllelicCounts = allelicCounts;
```

Comment:
Is there any reason you moved the declaration/definition up here?  You end up setting it a couple lines later, so it doesn't seem to make a difference.

Reply:
Just a very minor matter of style.  Now all subsequent transformations after the initial declaration operate on the new variable, so any transformations added later will have an identical format.  (Really what happened is that I experimented with adding filtering steps and changing their order, but got bit by not noticing I had inadvertently reverted to the original counts in a later step due to a careless copy and paste...)